languages/woocommerce-myparcel-fr_BE.po

```diff
@@ -180,8 +180,15 @@ msgid "Carrier"
 msgstr "Transporteur"
 
 #: includes/admin/ShipmentOptions.php:106
+#: includes/admin/class-wcmypa-admin.php:937
+#: includes/admin/class-wcmypa-admin.php:952
```

Comment:
is dit met gulp gedaan?

Reply:
Nee in master zit geen gulp, dit is met loco translate gedaan